pepper-apis/src/main/java/org/broadinstitute/ddp/security/AesUtil.java

```diff
@@ -159,10 +159,12 @@ public static void main(String[] args) throws Exception {
 
         if (cmd.hasOption('d')) {
             System.out.println(AesUtil.decrypt(cmd.getOptionValue('s'), encryptionSecret));
+            return;
```

Comment:
Each flag in AesUtil does one thing, so we return after finishing that one thing.